the-forest.kvp

```diff
@@ -1,35 +1,87 @@
-App.AdminMethod=STDIO
-App.ApplicationPort1=27015
-App.ApplicationPort2=8766
-App.ApplicationPort3=27016
-App.ApplicationReadyMode=Immediate
-App.BaseDirectory=./the-forest/556450/
-App.LinuxCommandLineArgs=run {{$ExecutableWin}} 
-App.CommandLineArgs={{$PlatformArgs}}-showlogs -batchmode -configfilepath server.cfg -savefolderpath saves -serverip {{$ApplicationIPBinding}} -servergameport {{$ApplicationPort1}} -serversteamport {{$ApplicationPort2}} -serverqueryport {{$ApplicationPort3}} -serverplayers {{$MaxUsers}} {{$FormattedArgs}}
-App.CommandLineParameterDelimiter= 
-App.CommandLineParameterFormat=-{0} "{1}"
+Meta.DisplayName=The Forest
+Meta.Description=The Forest Dedicated Server
+Meta.OS=Windows, Linux
+Meta.Arch=x86_64
+Meta.Author=CubeCoders Limited, IceOfWraith
+Meta.URL=https://store.steampowered.com/app/242760/The_Forest/
+Meta.DisplayImageSource=steam:242760
+Meta.EndpointURIFormat=steam://connect/{ip}/{GenericModule.App.ApplicationPort3}
```

Comment:
Format needs to be fixed - colon and new port format

Reply:
This has been corrected.